website/docs/components/form/masked-input/partials/code/component-api.md

```diff
@@ -0,0 +1,109 @@
+    <br/><br/>
+    Examples of HTML attributes: `id`, `name`, `value`, `placeholder`, `disabled`, `readonly`, `required`. See [the whole list of HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes). Examples of Ember modifiers: `\{{on "input" [do something]}}`, `\{{on "change" [do something]}}`, `\{{on "blur" [do something]}}`.
+  </C.Property>
+</Doc::ComponentApi>
+
+### Form::MaskedInput::Field
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="value" @type="string|number|date">
+    Input control’s value.
+  </C.Property>
+  <C.Property @name="isMasked" @type="boolean" @default="true">
+    Set this argument to `false` to make the input content visible by default or bind it to a variable to control the masking from outside the component.
+  </C.Property>
+  <C.Property @name="isMultiline" @type="boolean" @default="false">
+    By default, the component renders an `<input>`. If set to `true` it renders a `<textarea>`.
+  </C.Property>
+  <C.Property @name="isInvalid" @type="boolean" @default="false">
+    Applies an “invalid” appearance to the control but doesn’t modify its logical validity.
+  </C.Property>
+  <C.Property @name="isRequired" @type="boolean" @default="false">
+    Appends a `Required` indicator next to the label text and sets the `required` attribute on the control when user input is required.
+  </C.Property>
+  <C.Property @name="isOptional" @type="boolean" @default="false">
+    Appends an `Optional` indicator next to the label text when user input is optional.
+  </C.Property>
+  <C.Property @name="width" @type="string" @valueNote="any valid CSS width (px, rem, etc)">
+    By default, the input fills the parent container. If a `@width` parameter is provided, the control will have a fixed width. This width will only be applied to the control, not the other elements of the field.
+  </C.Property>
+  <C.Property @name="height" @type="string" @valueNote="any valid CSS height (px, rem, etc)">
+    Only available if `@IsMultiline` is `true`. By default, the `<textarea>` has a `height` determined by the browser to accommodate 4 lines of text. If a custom `@height` value is provided, the control will have a fixed height.
+  </C.Property>
+  <C.Property @name="id" @type="string">
+    Input control’s ID attribute.
+    <br/><br/>
+    By default, the ID is automatically generated by the component. Use this argument to pass a custom ID.
+  </C.Property>
+  <C.Property @name="extraAriaDescribedBy" @type="string">
+    Extra ID attribute to add to the `aria-describedby` HTML attribute.
+    <br/><br/>
+    By default, the `aria-describedby` attribute is automatically generated by the component, using the IDs of the helper text and errors (if present). Use this argument to pass an extra ID.
```

Comment:
Does it really pass an extra ID or does it pass a custom ID value vs. using the generated value?